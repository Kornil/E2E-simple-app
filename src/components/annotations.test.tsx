import React from "react";
import { render } from "@testing-library/react";

import Annotations from "./annotations";

jest.mock("uuid/v4", () => () => "test-id");

describe("<Annotations />", () => {
  it("matches snapshots", () => {
    const { container } = render(<Annotations />);
    expect(container).toMatchSnapshot();
  });
});
