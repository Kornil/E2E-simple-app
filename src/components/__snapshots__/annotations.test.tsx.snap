// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Annotations /> matches snapshots 1`] = `
.emotion-0 {
  width: 100vw;
  height: 100vh;
  background: #fff;
  position: relative;
  font-family: "Quicksand",sans-serif;
}

<div>
  <div
    class="emotion-0 emotion-1"
  />
</div>
`;
